Source/CronosPlayUnreal/Private/PlayCppSdkActor.cpp

```diff
@@ -649,9 +711,16 @@ void APlayCppSdkActor::SendEip155Transaction(
     // if no address, return
     if (address.Num() == 0 || chain_id == 0)
         return;
+
+    TWeakObjectPtr<APlayCppSdkActor> WeakThis(this);
     AsyncTask(ENamedThreads::AnyHiPriThreadNormalTask, [Out, coreclient,
                                                         address, chain_id, info,
-                                                        this]() {
+                                                        WeakThis]() {
+        if (!WeakThis.IsValid()) {
```

Comment:
This condition is evoked multiple times. Maybe we can create a helper funtion. Something like

Example
```cpp
handleIfValid(weakThis, [](auto obj) {
    // Code that requires a valid object
    obj->someMethod();
});
```

And then just invoke that function.

Reply:
ok, i'll check